linux/dev_setup_script.sh

```diff
@@ -0,0 +1,84 @@
+printf "\n\n====> Linux Development Setup <===="
+printf "\nEach group of packages wl be install one at a time. Follow the prompts to continue.\n"
```

Comment:
Please use the same log function that is used in the other scripts in this repo.